packages/bot/src/stores/flyout-store.js

```diff
@@ -0,0 +1,202 @@
+/* eslint-disable no-underscore-dangle */
+import React                  from 'react';
+import { observable, action } from 'mobx';
+import FlyoutBlock            from '../components/flyout-block.jsx';
```

Comment:
@aaron-binary you don't need to import component here? you should send an array to your components and create a subcomponent and map over your array to create anything you need.